Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up
+
+import random
+
+#These are my constants for the future I will UPPER_SNAKE_CASE
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 'Nine':9, 'Ten':10, 'Jack':10,
+         'Queen':10, 'King':10, 'Ace':11}

```

Comment:
Formatting: There should be a space before the value in `key: value` pairs.
Again, not that important here, but these are examples of things I would point out in reviews.

Sometime in the future we can talk about how to setup a "linter" that can highlight all the formatting stuff and even auto-format a lot of it.

Reply:
Good point PEP8 style was not kept up here.

Reply:
also, its interesting how Pycharm suggested the fix.